### PR TITLE
@kanaabe => Update React to 16

### PR DESCRIPTION
**Pull request by damassi**
Follows in line with:
- https://github.com/artsy/positron/pull/1510
- https://github.com/artsy/reaction/pull/273

I've done some clicking around but would appreciate if you could pull it down and run! Note though that without reaction/542 up above [this page](http://localhost:5000/gene/20th-century-furniture-and-design) will throw this error: 

screen shot 2017-12-20 at 6 19 55 pm

### CODE REVIEW

**Comment by alloy** [comment]
Reaction has been updated to 0.17, so this PR should pull that in.

File: package.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
   },
   "dependencies": {
     "@artsy/express-reloadable": "^1.1.0",
-    "@artsy/reaction-force": "0.16.13",
-    "@artsy/stitch": "^1.2.1",
+    "@artsy/reaction-force": "0.16.14",
```
Comment (alloy) [inline package.json]: This should be bumped to 0.17

**Comment by kanaabe** [comment]
I pulled this down, upgraded to 0.17.1 locally and checked out a bunch of pages. articles, genes, latest gucci/bmw stuff all looks great! I think this is ready to go once you grab 0.17.1!

**Comment by damassi** [comment]
Yeahhhh awesome, updating now

**Pull request merged**